dev/com.ibm.ws.jdbc_fat_derby/fat/src/com/ibm/ws/jdbc/fat/derby/JDBCDerbyTest.java

```diff
@@ -48,6 +62,147 @@ public static void setUp() throws Exception {
 
     @AfterClass
     public static void tearDown() throws Exception {
-        server.stopServer("CWWKE0701E"); //expected by testTransactionalSetting
+        server.stopServer("CWWKE0701E", //expected by testTransactionalSetting
+                          "DSRA4011E", //expected by testTNConfigIsoLvlReverse
+                          "SRVE0319E"); //expected by testTNConfigTnsl
+    }
+
+    /**
+     * Update the data source configuration while the server is running.
+     * Ensure that when using a data source with isolation level of TRANSACTION_NONE
+     * that when switching to a data source with Transactional = true the change is prevented
+     */
+    @Test
+    @ExpectedFFDC({ "com.ibm.wsspi.injectionengine.InjectionException", "javax.servlet.UnavailableException", "java.sql.SQLException" })
+    public void testTNConfigTnsl() throws Throwable {
+        String method = "testTNConfigTnsl";
+        Log.info(c, method, "Executing " + method);
+
+        //First check that we are not enlisting in transactions
+        runTest(server, jdbcappfat, "testTNTransationEnlistment");
+
+        ServerConfiguration config = server.getServerConfiguration();
+        DataSource ds12 = config.getDataSources().getBy("id", "dsfat12");
+
+        try {
+            ds12.setTransactional("true");
+
+            //Update config
+            server.setMarkToEndOfLog();
+            server.updateServerConfiguration(config);
+            server.waitForConfigUpdateInLogUsingMark(new TreeSet<String>(Arrays.asList(jdbcapp)));
+
+            //Ensure that updating the config resulted in server not being able to restart
+            int responseCode = runTestForResponseCode(server, jdbcappfat, "testTNTransationEnlistment");
```

Comment:
Does this really result in the server not starting?  That is not what we want...